tools/sidekick/config.json

```diff
@@ -1,4 +1,6 @@
 {
+  "project": "CN Web",
+  "host": "main--cn-website--netcentric.hlx.page/",
```

Comment:
are you sure we need this?

Reply:
We can remove the project for sure but I am not so sure about removing the host